server/api/[server]/oauth/[origin].ts

```diff
@@ -24,6 +26,9 @@ export default defineEventHandler(async (event) => {
   try {
     const result: any = await $fetch(`https://${server}/oauth/token`, {
       method: 'POST',
+      headers: {
+        'user-agent': APP_NAME,
```

Comment:
I guess we should append the version as well, `Elk/0.x.x`. Could put in the constants module.

I am not sure if we should also include other UA for compatibility, otherwise it LGTM

Reply:
what changes need to be done to the release process so that this constant gets updated?

Reply:
You can get the version here https://github.com/elk-zone/elk/blob/57814915d655acecb578718a99d602218e0c3fe7/config/env.ts#L4

Reply:
ah okay, ty. i thought you were suggesting to add a new constant. will fix soon

Reply:
> I am not sure if we should also include other UA for compatibility, otherwise it LGTM

I believe this should not be necessary, considering you got away with literally hte string `"undici"` so far